exporters/InventorRobotExporter/InventorRobotExporter/Exporter/SkeletonBuilder.cs

```diff
@@ -38,6 +38,7 @@ public static RigidNode_Base ExportSkeleton(IProgress<ProgressUpdate> progress)
             catch (NoGroundException)
             {
                 WinFormsUtils.ShowErrorDialog("Please ground a part in your assembly to export your robot.", "No Ground");
+                RobotExporterAddInServer.Instance.RobotDataManager.hasGround = false;
```

Comment:
Could you make this a generic `wasForceQuit` flag and have it enable in the force quit function so this works with other cases where the environment is force quit?

Reply:
Sure